docs/advanced_topics/customisation/page_editing_interface.md

```diff
@@ -13,9 +13,9 @@ class BlogPage(Page):
     # field definitions omitted
 
     content_panels = [
-        FieldPanel('title', classname="full title"),
+        FieldPanel('title', classname="title"),
         FieldPanel('date'),
-        FieldPanel('body', classname="full"),
+        FieldPanel('body'),
```

Comment:
does this mean that `FieldPanel` will no longer support an arbitrary classname?

Reply:
Spoke with Thibaud
* Still some examples that use the classname, only full is going away
* Code still supports this.